src/utils/useTxMulti/useTxMulti.ts

```diff
@@ -30,6 +34,28 @@ export type TUseTxMulti = () => {
 export const useTxMulti: TUseTxMulti = () => {
   const [state, dispatch] = useReducer(TxMultiReducer, initialState, identity, 'TxMulti');
   const getState = () => state;
+  const { accounts } = useContext(StoreContext);
+  const { addNewTxToAccount } = useContext(AccountContext);
+  const { assets } = useAssets();
+  const { getNetworkByChainId } = useNetworks();
+  const { account } = state;
+
+  const currentTx: TxParcel = view(lensIndex(state._currentTxIdx), state.transactions);
+
+  useEffect(() => {
+    if (
+      account &&
+      currentTx &&
+      currentTx.txResponse &&
+      currentTx.txHash &&
+      currentTx.status === ITxStatus.BROADCASTED
+    ) {
+      const network = getNetworkByChainId(currentTx.txRaw.chainId)!;
+      const txConfig = makeTxConfigFromTxResponse(currentTx.txResponse, assets, network, accounts);
+      const pendingTxReceipt = makePendingTxReceipt(currentTx.txHash)(ITxType.UNKNOWN, txConfig);
+      addNewTxToAccount(account, pendingTxReceipt);
```

Comment:
It's really odd that we need so many manipulations between formats.
Can me converge the them?

Reply:
I think this is a general thing that we have to look into. But very much out of the scope of this PR.